metricbeat/metricbeat.full.yml

```diff
@@ -238,12 +238,29 @@ metricbeat.modules:
   hosts: ["localhost:5601"]
 
 #----------------------------- kubernetes Module -----------------------------
+# Node metrics, from kubelet:
 - module: kubernetes
-  metricsets: ["node","container","volume","pod","system"]
   enabled: false
+  metricsets:
+    - node
+    - system
+    - pod
+    - container
+    - volume
   period: 10s
   hosts: ["localhost:10255"]
 
+# State metrics from kube-state-metrics service:
+- module: kubernetes
+  enabled: false
+  metricsets:
+    - state_node
```

Comment:
Based on a comment from @andrewkroh in an other PR I'm hoping we can make this state.node etc. in the future (not in this PR).